etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/ruleengine/Rule.java

```diff
@@ -0,0 +1,25 @@
+package gov.hhs.cdc.trustedintermediary.etor.ruleengine;
+
+import java.util.List;
+import org.hl7.fhir.instance.model.api.IBaseResource;
```

Comment:
So... this is a Hapi FHIR import.  This means this validation framework is tightly coupled to the Hapi FHIR library.

1. Do we want that?
2. Could we add some abstraction to this?
3. Maybe we just rename all of these classes by prepending `Hapi`?

See what we wrote in ADR 13 (`/adr/013-wrapped-domain-objects.md`).

I'm not pushing you to a specific answer, and I want to know what your thoughts are on this.

Reply:
Good catch. I had pending task that I missed to review all the places where we were using the HapiFHIR library to make sure we decouple the dependency. I'll work on refactoring it

Reply:
I refactored the code to decouple the hapifhir dependency